src/main/msp/msp.c

```diff
@@ -626,6 +627,10 @@ static bool mspCommonProcessOutCommand(int16_t cmdMSP, sbuf_t *dst, mspPostProce
         targetCapabilities |= (getRxBindSupported() << TARGET_SUPPORTS_RX_BIND_BIT);
 #endif
 
+#if defined(USE_ACC)
+        targetCapabilities |= (!accHasBeenCalibrated() << TARGET_ACC_NEEDS_CALIBRATION_BIT);
```

Comment:
I suppose that the decision if the accelerometer is needed is done in other place. If I'm not wrong here we always send we need to calibrate it, despite if it's enabled or not.

Reply:
Well spotted - checking if ACC is enabled has fallen through the cracks in my first attempt. The idea is to only report calibration yes / no here, and then use the information from `MSP_STATUS` in configurator to check if ACC is enabled.

I've updated betaflight/betaflight-configurator#1903 to do this.